pkg/client/default_producer.go

```diff
@@ -49,7 +47,3 @@ func (p *DefaultProducer) ForDynamic() dynamic.Interface {
 func (p *DefaultProducer) ForSubmariner() submarinerclient.Interface {
 	return p.SubmarinerClient
 }
-
-func (p *DefaultProducer) ForOperator() operatorclient.Interface {
```

Comment:
**subctl** uses this.

Reply:
Perhaps we need a test for subm-op to make sure were not breaking subctl